specification/cloudevents.md

```diff
@@ -288,6 +372,33 @@ The *start evaluation* event is sent when a test for a service is completed and
   }
 }
 ```
+
+Example for evaluation event during canary release:
+
+```json
+{
+  "type": "sh.keptn.event.evaluation.start",
+  "specversion": "0.2",
+  "source": "https://github.com/keptn/keptn/kernel",
+  "id": "49ac0dec-a83b-4bc1-9dc0-1f050c7e781b",
+  "time": "2019-06-07T07:02:15.64489Z",
+  "contenttype": "application/json",
+  "shkeptncontext":"49ac0dec-a83b-4bc1-9dc0-1f050c7e789b",
+  "shkeptnphaseid":"k5dg-565h-9o87",
+  "shkeptnphase":"RELEASE",
+  "shkeptnstepid":"a3ce-grdv-qwed",
+  "shkeptnstep":"TEST",
+  "data": {
+    "stage": "production",
+    "project": "sockshop",
+    "service": "carts",
+    "teststrategy":"real-user",
+    "startedat": "timestamp",
```

Comment:
let's provide an example value here as well, to indicate that it should be in the same format as the `time` property of the cloud event, e.g. `2019-06-07T07:02:15.64489Z`

Reply:
thanks for the hint!